src/sandstorm/spk.c++

```diff
@@ -889,11 +889,99 @@ private:
+        "      # https://github.com/sandstorm-io/sandstorm/blob/master/src/sandstorm/package.capnp\n"
+        "\n"
+        "      author = (\n"
+        "        # Fields relating to the author of this app.\n"
+        "\n"
+        "        contactEmail = \"youremail@example.com\",\n"
+        "        # Email address to contact for any issues with this app. This includes end-user support\n"
+        "        # requests as well as app store administrator requests, so it is very important that this be a\n"
+        "        # valid address with someone paying attention to it.\n"
+        "\n"
+        "        pgpSignature = embed \"path/to/pgp-signature\",\n"
+        "        # PGP signature attesting responsibility for the app ID. This is a binary-format detached\n"
+        "        # signature of the following ASCII message (not including the quotes, no newlines, and\n"
+        "        # replacing <app-id> with the standard base-32 text format of the app's ID):\n"
+        "        #\n"
+        "        # \"I am the author of the Sandstorm.io app with the following ID: <app-id>\"\n"
+        "        #\n"
+        "        # You can create a signature file using `gpg` like so:\n"
+        "        #\n"
+        "        #     echo -n \"I am the author of the Sandstorm.io app with the following ID: <app-id>\" | gpg --sign > pgp-signature\n"
+        "        #\n"
+        "        # To learn how to set up gpg, visit Keybase (https://keybase.io) -- they have excellent\n"
+        "        # documentation and tools. Moreover, if you create a Keybase account for your key and follow\n"
+        "        # Keybase's instructions to link it to other social accounts (like your Github account), then\n"
+        "        # the Sandstorm app install flow and app market will present this information to the user as\n"
+        "        # \"verified\".\n"
+        "\n"
+        "        #upstreamAuthor = \"Example App Team\",\n"
+        "        # Name of the original primary author of this app, if it is different from the person who\n"
+        "        # produced the Sandstorm package. Setting this implies that the author connected to the PGP\n"
+        "        # signature only \"packaged\" the app for Sandstorm, rather than developing the app.\n"
+        "      ),\n"
+        "      pgpKeyring = embed \"path/to/pgp-keyring\",\n"
+        "      # A keyring in GPG keyring format containing all public keys needed to verify PGP signatures in\n"
+        "      # this manifest (as of this writing, there is only one: `author.pgpSignature`).\n"
+        "      #\n"
+        "      # To generate a keyring containing just your public key, do:\n"
+        "      #\n"
+        "      #     gpg --export <key-id> > keyring\n"
+        "      #\n"
+        "      # Where `<key-id>` is a PGP key ID or email address associated with the key.\n"
+        "\n"
+        "      description = (defaultText = embed \"path/to/description.md\"),\n"
```

Comment:
I think we should auto-generate a `.sandstorm/app-market-description.md` and then refer to it here, like what GitHub does for `README.md` on new repositories.

Else comment out.